spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -573,7 +577,7 @@ def compute_all_job_data(
     ) -> SparkApplication:
         app_model = raw_data
         t1 = time.time()
-        dfs: list[pd.DataFrame] = []
+        dfs: list[pd.DataFrame] = [pd.DataFrame()]
```

Comment:
Although this works, I think it would be better to address the "jobless" scenario more explicitly because as is, it's just not obvious what purpose the initialization serves. As an alternative, you might do the following below:

```
if dfs:
   df = pd.concat(dfs)
else:
   df = pd.DataFrame()
```